apps/heft/src/cli/actions/TestAction.ts

```diff
@@ -1,40 +1,115 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
```

Comment:
1. In the `heft-test-01` project folder, run `heft test --clean`
2. Rename **ExampleTest.test.ts** to **ExampleTest2.test.ts**
3. Run `heft test --clean --production`

**Expected:** The build should fail because the snapshot file is missing.
**Actual:** Heft automatically updates the snapshot.  How do we get CI to break if the snapshots are out of date?

3. Run `heft test --clean --production` again

**Expected:** The ExampleTest.test.js.snap file should get deleted by now, or report a warning of some kind.
**Actual:** It sticks around as deadwood, and the build succeeds.

`jest --watch` automatically deletes the obsolete snapshot.  But `heft --watch` does not.